src/utils.py

```diff
@@ -6,62 +6,45 @@
 """
 import logging
 import subprocess
+from pathlib import Path
 
 logging.getLogger().setLevel(logging.INFO)
 
 
 class DataHandler:
-    def __init__(self, data_path, dataset_name):
+    def __init__(self, data_path):
         self.data_path = data_path
-        self.dataset_name = dataset_name
         logging.info(f"Data Handler Class Created - {data_path}")
-        if dataset_name == "Indosum":
-            self.download_indosum()
 
-    def download_indosum(self):
+    def downloader(self, download_type):
         """
-        Downaloads Indosum in data_path/indosum.
+        Downaloads download_type in data_path/download_type.
         """
-        logging.info("Downloading Indosum")
-        if not self.check_data_exists(self.dataset_name):
-            (self.data_path / self.dataset_name).mkdir(parents=True, exist_ok=True)
-        else:
-            logging.info("Folder already exists")
-        if not (self.data_path / self.dataset_name / "indosum.tar.gz").exists():
+        logging.info(f"Downloading {download_type}")
+        if not (self.data_path / f"{download_type}.tar.gz").exists():
+            google_drive_link = {
+                "indosum": "https://docs.google.com/uc?export=download&id=1OgYbPfXFAv3TbwP1Qcwt_CC9cVWSJaco",
+                "indo_lm": "https://drive.google.com/uc?export=download&id=1ez8QfKhAK5tL41usBq8K1mUQAUBjB7R1",
+            }
             output_download = subprocess.check_output(
                 [
                     "wget",
                     "--no-check-certificate",
-                    "https://docs.google.com/uc?export=download&id=1OgYbPfXFAv3TbwP1Qcwt_CC9cVWSJaco",
+                    google_drive_link[download_type],
                     "-O",
-                    f"{self.data_path/self.dataset_name/'indosum.tar.gz'}",
+                    f"{self.data_path/f'{download_type}.tar.gz'}",
```

Comment:
```
                    f"{self.data_path/f'{download_type}.tar.gz'}",
```

This is hard to read. Simplify by creating a variable like: 
```
fname = f"{download_type}.tar.gz"
```
and then this statement can be something like:
```
f"{self.data_path / fname}"
```